scripts/create-clusters.sh

```diff
@@ -123,18 +124,21 @@ function fixup-cluster() {
   # TODO(font): Need to set container IP address in order for clusters to reach
   # kube API servers in other clusters until
   # https://github.com/kubernetes-sigs/kind/issues/111 is resolved.
-  local container_ip_addr=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kind-${i}-control-plane)
-  sed -i "s/localhost/${container_ip_addr}/" ${kubeconfig_path}
-
+  if [ "$OS" != "Darwin" ];then
+      local container_ip_addr=$(docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kind-${i}-control-plane)
+      sed -i.bak "s/localhost/${container_ip_addr}/" ${kubeconfig_path} && rm -rf ${kubeconfig_path}.bak
+  fi
   # TODO(font): Need to rename auth user name to avoid conflicts when using
   # multiple cluster kubeconfigs. Remove once
   # https://github.com/kubernetes-sigs/kind/issues/112 is resolved.
-  sed -i "s/kubernetes-admin/kubernetes-kind-${i}-admin/" ${kubeconfig_path}
+  # Compatible with MacOS
```

Comment:
i guess majority of the audience is linux-only folks, who are not necessarily familiar with non-GNU sed implementations.
`GNU sed does .... on the other hand, BSD sed found on OSX does .... this invocation is intended to be compatible with both of them`
maybe you can create a function to hide the difference.

btw, talking about sed, haven't you seen problems with whitespace treatment difference in "a" command used in various scripts?

Reply:
yeah.  But case by case, no whitespace is need to be treated here.
Do you really think it worth the effort that we wrap sed here?